mne/viz/_brain/_timeviewer.py

```diff
@@ -491,6 +505,37 @@ def set_slider_style(self, slider, show_label=True):
             if not show_label:
                 slider_rep.ShowSliderLabelOff()
 
+    def help(self):
+        shortcuts = {
+            'h': 'Display help window',
+            'y': 'Toggle interface',
+            't': 'Apply auto-scaling',
+            'u': 'Restore original clim',
+            'Space': 'Start/Pause playback'
+        }
+        text = [str(s) + " : \n" for s in shortcuts.keys()]
+        text2 = [str(s) + "\n" for s in shortcuts.values()]
+        text, text2 = ''.join(text), ''.join(text2)
+        width, height = 9, 5
+
+        fig_help = figure_nobar(figsize=(width, height), dpi=80)
+        fig_help.canvas.set_window_title('Help')
+
+        ax = fig_help.add_subplot(111)
+        celltext = [[c1, c2] for c1, c2 in zip(text.strip().split("\n"),
+                                               text2.strip().split("\n"))]
+        table = ax.table(cellText=celltext, loc="center", cellLoc="left")
+        table.auto_set_font_size(False)
+        table.set_fontsize(12)
+        ax.set_axis_off()
+        for (row, col), cell in table.get_celld().items():
+            cell.set_edgecolor(None)  # remove cell borders
+            if col == 0:
+                cell._loc = 'right'
+
+        fig_help.canvas.draw()
+        plt_show(fig=fig_help, warn=False)
```

Comment:
Please refactor this code with the other help windows in `mne/viz`. They should all use a single call, maybe something like:
```
def _show_help(options):
    ...
```
where `options` is a list (rows) of 2-element lists (cols)? This would be better than a `dict` because you make it so that some of the "rows" are actually blank, or hline, etc. This function would take that input and pop up a window showing the table, this window should be close-able with hitting `esc`, etc.